packages/plugin-connector/src/data-sources/get-sources.js

```diff
@@ -163,9 +163,61 @@ async function loadConnectionOptions(sourceDir) {
  * containing the filepath and content of each query file.
  */
 async function getQueries(sourceDir, contents) {
-	const queryFiles = contents.filter(
-		(s) => s !== 'connection.yaml' && s !== 'connection.options.yaml'
+	const queryFiles = await Promise.all(
+		contents
+			.filter((s) => s !== 'connection.yaml' && s !== 'connection.options.yaml')
+			.flatMap(
+				/**
+				 * @param {string} s
+				 * @returns {Promise<string[]>}
+				 */
+
+				async (s) => {
```

Comment:
IMO clearer name would be better, especially when it's ambiguously a dir or file name - also update the jsdoc for `contents` to say array of file names or directories